front/lib/api/assistant/actions/visualization.ts

```diff
@@ -189,14 +193,39 @@ export class VisualizationConfigurationServerRunner extends BaseActionConfigurat
       action: new VisualizationAction({
         id: action.id,
         agentMessageId: action.agentMessageId,
-        query,
         output: null,
         functionCallId: action.functionCallId,
         functionCallName: action.functionCallName,
         step: action.step,
       }),
     };
 
+    // Turn the conversation into a digest that can be presented to the model.
+    const MIN_GENERATION_TOKENS = 2048;
+    const agentModelConfig = getSupportedModelConfig(agentConfiguration.model);
+    const modelConversationRes = await renderConversationForModelMultiActions({
+      conversation,
+      model: agentModelConfig,
+      prompt: "", // There is no prompt for title generation.
+      allowedTokenCount: agentModelConfig.contextSize - MIN_GENERATION_TOKENS,
+      excludeActions: true,
```

Comment:
Can you provide some context on the rationale of excluding actions here ? 
I think previous code, retrievals etc.. could be valuable to generate code no ?

Reply:
No good reason, cargo culted, I'll update!